scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/client/SchedulerNodeClient.java

```diff
@@ -0,0 +1,436 @@
+        return client.getTaskIds(taskTag, from, to, mytasks, running, pending, finished, offset, limit);
+    }
+
+    public Page<TaskState> getTaskStates(String taskTag, long from, long to, boolean mytasks, boolean running, boolean pending, boolean finished, int offset, int limit, SortSpecifierContainer sortParams) throws NotConnectedException, PermissionException {
+        return client.getTaskStates(taskTag, from, to, mytasks, running, pending, finished, offset, limit, sortParams);
+    }
+
+    public JobInfo getJobInfo(String jobId) throws UnknownJobException, NotConnectedException, PermissionException {
+        return client.getJobInfo(jobId);
+    }
+
+    public boolean changeStartAt(JobId jobId, String startAt) throws NotConnectedException, UnknownJobException, PermissionException {
+        return client.changeStartAt(jobId, startAt);
+    }
+
+    public JobId copyJobAndResubmitWithGeneralInfo(JobId jobId, Map<String, String> generalInfo) throws NotConnectedException, UnknownJobException, PermissionException, SubmissionClosedException, JobCreationException {
+        return client.copyJobAndResubmitWithGeneralInfo(jobId, generalInfo);
+    }
+
+    public void init(ConnectionInfo connectionInfo) throws Exception {
+        client.init(connectionInfo);
+    }
+
+    public void setSession(String sid) {
+        client.setSession(sid);
+    }
+
+    public String getSession() {
+        return client.getSession();
+    }
+
+    public boolean isJobFinished(JobId jobId) throws NotConnectedException, UnknownJobException, PermissionException {
+        return client.isJobFinished(jobId);
+    }
+
+    public boolean isJobFinished(String jobId) throws NotConnectedException, UnknownJobException, PermissionException {
+        return client.isJobFinished(jobId);
+    }
+
+    public JobResult waitForJob(JobId jobId, long timeout) throws NotConnectedException, UnknownJobException, PermissionException, TimeoutException {
+        return client.waitForJob(jobId, timeout);
+    }
+
+    public JobResult waitForJob(String jobId, long timeout) throws NotConnectedException, UnknownJobException, PermissionException, TimeoutException {
+        return client.waitForJob(jobId, timeout);
+    }
+
+    public boolean isTaskFinished(String jobId, String taskName) throws UnknownJobException, NotConnectedException, PermissionException, UnknownTaskException {
+        return client.isTaskFinished(jobId, taskName);
+    }
+
+    public TaskResult waitForTask(String jobId, String taskName, long timeout) throws UnknownJobException, NotConnectedException, PermissionException, UnknownTaskException, TimeoutException {
+        return client.waitForTask(jobId, taskName, timeout);
+    }
+
+    public List<JobResult> waitForAllJobs(List<String> jobIds, long timeout) throws NotConnectedException, UnknownJobException, PermissionException, TimeoutException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1161)